imsv-docs-astro/src/content/docs/guides/core-concepts/card-payments-101.mdoc

```diff
@@ -1,5 +1,44 @@
 ---
 title: Card Payments 101
+slug: guides/card-payments-101
 sidebar:
-  hidden: true
+  order: 1
 ---
+
+
+Immersve facilitates payments on the Mastercard<sup>®</sup> network by
+implementing the required card payment protocols. The main protocol
+instructions are: Authorization, Clearing, and Settlement.
+
+{% note %}
+Integrators can test card payment protocols with the Immersve payment
+simulator. The payment simulator uses test assets and a mock payment network.
```

Comment:
Might be good to add a link to payment simulator docs

Reply:
Haha nice catch. My link for stripped when I pasted back from Hemingway